functions/go/format/main.go

```diff
@@ -3,7 +3,9 @@ package main
 import (
 	"fmt"
 	"os"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework/command"
```

Comment:
Can you please group std lib in one group and other lib in a separate group?

Reply:
Done.